Python.wiki/Sorting-Algorithms.md

```diff
@@ -0,0 +1,93 @@
+<h1>index</h1>
+
+<br/>
+
+<ol>
+<li><a href="">Selection Sort</a></li>
```

Comment:
I am a bit uncomfortable with an unsorted list of sorting algorithms.   Are these sorted alphabetically sorted for speed, or other?  DIRECTORY.md is machine-generated to keep algorithms up-to-date and sorted alphabetically.

Reply:
This is an like for like drop-in except for the corrected markdown. The existing pages algorithms were already sorted like this. Ultimately, I was just trying to fix the issue highlighted in #11227 which also has the unsorted list. It can be edited by a repo admin in <30seconds